src/StackHCI/custom/stackhci.ps1

```diff
@@ -953,12 +955,37 @@ function Install-Dependent-Module{
             Install-PackageProvider NuGet -Force | Out-Null
         }
 
-        Write-VerboseLog ("Installing Module: {0} version: {1}" -f $ModuleName,$ModuleVersion)
+        Write-VerboseLog ("Installing Module: {0}" -f $ModuleName)
 
-        Install-Module -Name $ModuleName  -RequiredVersion $ModuleVersion  -Force -AllowClobber -Repository 'PSGallery'
-        Import-Module -Name $ModuleName -RequiredVersion $ModuleVersion
+        Install-Module -Name $ModuleName -Force -AllowClobber -Repository 'PSGallery'
+        Import-Module -Name $ModuleName
         
-        Write-VerboseLog ("Successfully imported Module: {0} version: {1}" -f $ModuleName,$ModuleVersion)
+        Write-VerboseLog ("Successfully imported Module: {0}" -f $ModuleName)
+    }
+}
+
+function Check-DependentModules 
+{
+    param()
+
+    $missingDependentModules = [System.Collections.ArrayList]::new()
+
+    if (-Not (Get-InstalledModule -Name "Az.Accounts" -MinimumVersion $AzAccountsModuleMinVersion -ErrorAction SilentlyContinue))
+    {
+        Write-VerboseLog "Required module Az.Accounts (minimum version: $AzAccountsModuleMinVersion) is missing"
+        $missingDependentModules.Add("Az.Accounts (minimum version: $AzAccountsModuleMinVersion)") | Out-Null
+    }
+    else 
+    {
+        Write-VerboseLog "Found required module Az.Accounts"
+        Import-Module -Name "Az.Accounts"
```

Comment:
```suggestion
```
No need to import Az.Accounts exlicitly

Reply:
Importing for the cases where a user (in this case, our developers) are importing Az.StackHCI PowerShell script directly.